### PR TITLE
task: update spec with support to toggle tokenization

### PR DESCRIPTION
**Pull request by luca-gr4vy**
# Description

Adds specification for the PS tokenization toggle support.

Depends on: https://github.com/gr4vy/core-api/pull/725

Fixes: https://gr4vy.atlassian.net/browse/TA-1112

Screenshot(s):

`/payment-services`

Screenshot 2022-03-28 at 11 01 37

`/payment-options`

Screenshot 2022-03-28 at 12 33 45

`/payment-service-definition` & `/payment-service-definition/{payment_service_definition_id}`

Screenshot 2022-04-11 at 15 13 08

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own changes
- [x] I have run `yarn test` to make sure my changes pass all linters
- [x] I have pulled the latest changes from the upstream `main` branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

**Comment by luca-gr4vy** [comment]
@cbetta getting some warnings when running the linter locally, seemingly unrelated to these changes. Is that ok?

Screenshot 2022-03-28 at 10 56 10

**Comment by cbetta** [comment]
> @cbetta getting some warnings when running the linter locally, seemingly unrelated to these changes. Is that ok?
> 
> Screenshot 2022-03-28 at 10 56 10

Yes those are expected

File: reference/resources/payment_option.yaml [hunk 1]
```diff
@@ -86,3 +86,8 @@ properties:
                 - "JCB"
                 - "MASTERCARD"
                 - "VISA"
+
+  payment_method_tokenization_enabled:
```
Comment (thomas-gr4vy) [inline reference/resources/payment_option.yaml:90]: This hasn't been added to `/payment-options`. Instead, the value of the existing `can_store_payment_method` property takes `payment_method_tokenization_enabled` into account.

**Review by thomas-gr4vy — CHANGES_REQUESTED** [review]

File: reference/resources/payment_option.yaml (hunk 1 above)
Comment (luca-gr4vy) [reply reference/resources/payment_option.yaml]: thanks, removed

**Review by cbetta — APPROVED** [review]

**Review by thomas-gr4vy — APPROVED** [review]

**Pull request merged**